.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### Links to any relevant github issues:
+
+- [Issue description](http://example.com/org/repo/issues/411). Any other useful information.
+
+### What's in the the PR
+
+- [ ] A description of the changes proposed in the pull request.
+  - [ ] Any areas or issues reviewer should pay attention to? (comments in diff okay).
+- [ ] Update `/docs` to include, whenever possible, a new, suitable recommendation about how to use
```

Comment:
That's something I would actually expect to find in the CONTRIBUTING part rather than the PR template. I would rather know before submitting a PR that I should add documentation.